aiogram/webhook/aiohttp_server.py

```diff
@@ -131,6 +133,10 @@ async def resolve_bot(self, request: web.Request) -> Bot:
         """
         pass
 
+    @abstractmethod
+    def secret_verify(self, secret_token: str) -> bool:
```

Comment:
please change these words (a verb followed by a noun - means do "something" on "thing")

```suggestion
    def veryfy_secret(self, secret_token: str) -> bool:
```